docs/instrumentation.md

```diff
@@ -137,11 +141,53 @@ AnyCable also supports emitting real-time metrics to [StatsD](https://github.com
 For that, you must specify the StatsD server UDP host:
 
 ```sh
-anycable-go -statsd_host=localhost:8125
+anycable-go --statsd_host=localhost:8125
 ```
 
 Metrics are pushed with the `anycable_go.` prefix by default. You can override it by specifying the `statsd_prefix` parameter.
 
+```sh
+# HELP anycable_go_clients_num The number of active clients
+# TYPE anycable_go_clients_num gauge
+anycable_go.clients_num
+
+# HELP anycable_go.mem_sys_bytes The total bytes of memory obtained from the OS
+# TYPE anycable_go.mem_sys_bytes gauge
+anycable_go.mem_sys_bytes
+
+# HELP anycable_go.goroutines_num The number of Go routines
+# TYPE anycable_go.goroutines_num gauge
+anycable_go.goroutines_num
+
+# HELP anycable_go.rpc_pending_num The number of pending RPC calls
+# TYPE anycable_go.rpc_pending_num gauge
+anycable_go.rpc_pending_num
+
+# HELP anycable_go.clients_uniq_num The number of unique clients (with respect to connection identifiers)
+# TYPE anycable_go.clients_uniq_num gauge
+anycable_go.clients_uniq_num
+
+# HELP anycable_go.grpc_active_conn_num The number of active gRPC connections are established
+# TYPE anycable_go.grpc_active_conn_num gauge
+anycable_go.grpc_active_conn_num
+
+# HELP anycable_go.broadcast_streams_num The number of active broadcasting streams
+# TYPE anycable_go_broadcast_streams_num gauge
+anycable_go.broadcast_streams_num
+
+# HELP anycable_go.disconnect_queue_size The size of delayed disconnect
+# TYPE anycable_go.disconnect_queue_size gauge
+anycable_go.disconnect_queue_size
+```
+
+### Datadog
+
+All you need is install [Datadog Agent v6+](https://docs.datadoghq.com/agent) only.
+
+See:
+
+* [AnyCable + Datadog + Heroku](https://docs.anycable.io/deployment/heroku?id=datadog)
+
```

Comment:
Let's remove this section; it's irrelevant to our documentation.

A good thing to add here could be a link to the demo application PR with a StatsD+Datadog configured (we can install a Datadog Agent inside a dev docker container for simplicity).